proposals/0060-ct-storage.md

```diff
@@ -0,0 +1,496 @@
+## Native TSDB Support for Cumulative Created Timestamp (CT) (and Delta Start Timestamp (ST) on the way)
+
+* **Owners:**
+  * [`@bwplotka`](https://github.com/bwplotka)
+  * <[delta-type-WG](https://docs.google.com/document/d/1G0d_cLHkgrnWhXYG9oXEmjy2qp6GLSX2kxYiurLYUSQ/edit) members?>
+
+* **Implementation Status:** `Partially implemented`
+
+* **Related Issues and PRs:**
+  * [WAL](https://github.com/prometheus/prometheus/issues/14218), [PRW2](https://github.com/prometheus/prometheus/issues/14220), [CT Meta](https://github.com/prometheus/prometheus/issues/14217).
+  * [initial attempt for ct per sample](https://github.com/prometheus/prometheus/pull/16046)
+  * [rw2 proto change for ct per sample](https://github.com/prometheus/prometheus/pull/17036)
+  * Initial implementation
+    * [Appender](https://github.com/prometheus/prometheus/pull/17104)
+    * [chunkenc.Iterator](https://github.com/prometheus/prometheus/pull/17176)
+
+* **Other docs or links:**
+  * [PROM-29 (Created Timestamp)](https://github.com/prometheus/proposals/blob/main/proposals/0029-created-timestamp.md)
+  * [Delta type proposal](https://github.com/prometheus/proposals/pull/48), [Delta WG](https://docs.google.com/document/d/1G0d_cLHkgrnWhXYG9oXEmjy2qp6GLSX2kxYiurLYUSQ/edit)
+
+> TL;DR: We propose to extend Prometheus TSDB storage sample definition to include an extra int64 that will represent the cumulative created timestamp (CT) and, for the future delta temporality ([PROM-48](https://github.com/prometheus/proposals/pull/48)), a delta start timestamp (ST).
+> We propose introducing persisting CT logic behind a single flag `ct-storage`.
+> Once implemented, we propose to deprecate the `created-timestamps-zero-injection` experimental feature.
```

Comment:
```suggestion
> Once implemented, we propose to remove the `created-timestamps-zero-injection` experimental feature.
```

Do we need to go through a deprecation notice for feature flags? I have the impression that the point of feature flags is that they are safe to remove 🤔

Reply:
experimental features esp have no guarantee of stability, so that seems correct